### PR TITLE
[Enhancement] Update cte tpcds test

### PR DESCRIPTION
**Pull request by Seaven**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

logic change
* update CTEAnchor costs compute

other, test relate:
* Support complete TPCDS-99 SQL
* Add 1T TPCDS statistics mock
* Add CTE Test on 1T TPCDS statsitics
* move MockTpchStatistics to test directory

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/sql/Explain.java [hunk 1]
```diff
@@ -819,12 +829,13 @@ static void buildCostEstimate(StringBuilder sb, OptExpression optExpression, int
 
         if (optExpression.getStatistics().getColumnStatistics().values().stream()
                 .allMatch(ColumnStatistic::isUnknown)) {
-            buildOperatorProperty(sb, "Estimates: {" +
-                    "row: " + (long) optExpression.getStatistics().getOutputRowCount() +
+            buildOperatorProperty(sb, "Estimates: {cost: " + optExpression.getCost() +
```
Comment (HangyuanLiu) [inline fe/fe-core/src/main/java/com/starrocks/sql/Explain.java:832]: cost, relative to other attributes, is a secondary attribute and should be placed at the end

**Review by Youngwb — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 29 / 43 (67.44%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/Explain.java|0|14|00.00%|[382, 383, 606, 607, 608, 609, 621, 622, 623, 629, 636, 638, 834, 841]|
|:large_blue_circle:|com/starrocks/sql/optimizer/Optimizer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/OptExpression.java|4|4|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/CsvFileStatisticsStorage.java|11|11|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java|11|11|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/cost/CostModel.java|2|2|100.00%|[]|

**Review by kangkaisen — DISMISSED** [review]

**Comment by Seaven** [comment]
run starrocks_admit_test

**Review by Youngwb — DISMISSED** [review]

**Comment by Seaven** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 29 / 43 (67.44%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/Explain.java|0|14|00.00%|[382, 383, 606, 607, 608, 609, 621, 622, 623, 629, 636, 638, 834, 841]|
|:large_blue_circle:|com/starrocks/sql/optimizer/Optimizer.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/OptExpression.java|4|4|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/CsvFileStatisticsStorage.java|11|11|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java|11|11|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/cost/CostModel.java|2|2|100.00%|[]|

**Review by HangyuanLiu — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**